shared/templates/grub2_bootloader_argument/ansible.template

```diff
@@ -33,6 +33,7 @@
   ansible.builtin.shell:
     cmd: '/usr/bin/grub2-editenv - list | grep "kernelopts="'
   register: kernelopts
+  ignore_errors: yes
```

Comment:
Where should the kernel argument be added when `kernelopts` doesn't exist?

I haven't tested, but it seems the task below will not add the option, and there are no other tasks to handle this case.

Reply:
That's true. We need to add a new step to set the parameter when kernelopts is not present

Reply:
added in 945c02c44588950850beec46fa38d09f2abfd546